src/item/item.cpp

```diff
@@ -1044,7 +1046,7 @@ Item* Item::removeFromList(std::vector<Item*>& list, int removeCount) {
     // item is inside a container
     removeFromContainer(removeCount);
   }
-  helpers::remove(list, this);
+  helpers::remove<item::Item*>(list, this);
```

Comment:
Was there a reason to be explicit for `helpers::remove`?

Reply:
I was getting a "C++ no instance of function template matches argument list" error earier. But testing it again now without the explicit type, it seems to be fine. I can only assume my IDE glitched out